sdk/attestation/attestation/review/attestation.api.md

```diff
@@ -11,18 +11,12 @@ import { TokenCredential } from '@azure/core-auth';
 // @public
 export class AttestationAdministrationClient {
     constructor(endpoint: string, credentials: TokenCredential, options?: AttestationAdministrationClientOptions);
-    addPolicyManagementCertificate(pemCertificate: string, privateKey: string, certificate: string, options?: AttestationAdministrationClientOperationOptions): Promise<AttestationResponse<PolicyCertificatesModificationResult>>;
-    getPolicy(attestationType: AttestationType, options?: AttestationAdministrationClientOperationOptions): Promise<AttestationResponse<string>>;
-    getPolicyManagementCertificates(options?: AttestationAdministrationClientOperationOptions): Promise<AttestationResponse<AttestationSigner[]>>;
-    removePolicyManagementCertificate(pemCertificate: string, privateKey: string, certificate: string, options?: AttestationAdministrationClientOperationOptions): Promise<AttestationResponse<PolicyCertificatesModificationResult>>;
-    resetPolicy(attestationType: AttestationType, options?: AttestationAdministrationClientOperationOptions & {
-        privateKey?: string;
-        certificate?: string;
-    }): Promise<AttestationResponse<PolicyResult>>;
-    setPolicy(attestationType: AttestationType, newPolicyDocument: string, options?: AttestationAdministrationClientOperationOptions & {
-        privateKey?: string;
-        certificate?: string;
-    }): Promise<AttestationResponse<PolicyResult>>;
+    addPolicyManagementCertificate(pemCertificate: string, privateKey: string, certificate: string, options?: AttestationAdministrationClientPolicyCertificateOperationOptions): Promise<AttestationResponse<PolicyCertificatesModificationResult>>;
```

Comment:
should not `privateKey` and `certificate` be gone from the parameter list now they're part of the options bag?

Reply:
They are in the options bag for the Policy APIs, they are mandatory parameters for the PolicyCertificates APIs, so they're explicitly listed in the parameters.